modules/virt-about-virtio-scsi.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-virtio-scsi_{context}"]
+= About virtio-scsi
+
+In {VirtProductName}, each virtual machine (VM) has a `virtio-scsi` controller so that hot-plugged disks can use a `scsi` bus. The `virtio-scsi` controller overcomes the limitations of `virtio` while retaining its performance advantages. It is highly scalable and supports hot-plugging over 4 million disks.
+
+Regular `virtio` is not available for hot-plugged disks because it is not scalable: each `virtio` disk uses one of the limited PCI Express (PCIe) slots in the VM. PCIe slots are also used by other devices and must be reserved in advance, so slots might not be available on demand.
```

Comment:
```suggestion
Regular `virtio` is not available for hot plugged disks because it is not scalable: each `virtio` disk uses one of the limited PCI Express (PCIe) slots in the VM. PCIe slots are also used by other devices and must be reserved in advance, therefore slots might not be available on demand.
```
See IBMSG:  https://www.ibm.com/docs/en/ibm-style?topic=word-usage

Reply:
Thank you for catching that!